books/free-programming-books-es.md

```diff
@@ -129,8 +129,6 @@
 
 ### C
```

Comment:
Remove C from here as well as Index above. And rectify the spacing as per the [guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)